python/turbine_models/custom_models/sdxl_inference/unet.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2023 Nod Labs, Inc
```

Comment:
Looks like a lot of repeated code from sd_inference/unet.py. Can we combine the two?